### PR TITLE
Add `Naming/AsciiIdentifiers` rule

### PR DESCRIPTION
**Pull request by Sija**

### CODE REVIEW

File: src/ameba/rule/naming/ascii_identifiers.cr [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  class AsciiIdentifiers < Base
+    include AST::Util
+
+    properties do
+      description "Disallows non-ascii characters in identifiers"
+    end
+
+    MSG = "Identifier contains non-ascii characters"
+
+    def test(source, node : Crystal::Assign)
```
Comment (veelenga) [inline src/ameba/rule/naming/ascii_identifiers.cr:33]: there is also [MultiAssign](https://github.com/crystal-lang/crystal/blob/master/src/compiler/crystal/syntax/ast.cr#L875)

File: src/ameba/rule/naming/ascii_identifiers.cr [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+      if location && end_location
+        check_issue(source, location, end_location, node.name)
+      else
+        check_issue(source, node, node.name)
+      end
+    end
+
+    def test(source, node)
+      if node.responds_to?(:name) && (name = node.name)
```
Comment (veelenga) [inline src/ameba/rule/naming/ascii_identifiers.cr:51]: not sure if that is a good idea to visit all the nodes and invoke responds_to? on them. Can this slow down process?

File: src/ameba/rule/naming/ascii_identifiers.cr (hunk 1 above)
Comment (Sija) [reply src/ameba/rule/naming/ascii_identifiers.cr:33]: Multiple assignment is [not allowed](https://github.com/crystal-lang/crystal/blob/c25640d79bcbf16b015cf9439a862d4626973cf2/src/compiler/crystal/syntax/parser.cr#L150) for constants.

File: src/ameba/rule/naming/ascii_identifiers.cr (hunk 2 above)
Comment (straight-shoota) [reply src/ameba/rule/naming/ascii_identifiers.cr:51]: The meat of `responds_to` is a compile-time operation. At runtime it's just a type filter (like `if x.is_a?(Foo)`).

**Review by veelenga — APPROVED** [review]
LGTM

File: src/ameba/rule/naming/ascii_identifiers.cr (hunk 1 above)
Comment (Sija) [reply src/ameba/rule/naming/ascii_identifiers.cr:33]: Actually, I'll extend these checks to all identifiers, not only constants.

**Pull request merged**